prime-router/src/main/kotlin/fhirengine/engine/FHIREngine.kt

```diff
@@ -20,6 +20,7 @@ import java.time.OffsetDateTime
 const val elrConvertQueueName = "elr-fhir-convert"
 const val elrRoutingQueueName = "elr-fhir-route"
 const val elrTranslationQueueName = "elr-fhir-translate"
+const val elrSendQueueName = "send"
```

Comment:
Nit: this is also the queue that covid pipeline uses.  If we're adding a tech debt ticket to split them though can leave this for now

Reply:
Yep, see #13232